code/modules/vehicles/scooter.dm

```diff
@@ -24,19 +24,43 @@
 	if(buckled_mobs.len)
 		for(var/m in buckled_mobs)
 			var/mob/living/buckled_mob = m
-			switch(buckled_mob.dir)
-				if(NORTH)
-					buckled_mob.pixel_x = 0
-					buckled_mob.pixel_y = 4
-				if(EAST)
-					buckled_mob.pixel_x = -2
-					buckled_mob.pixel_y = 4
-				if(SOUTH)
-					buckled_mob.pixel_x = 0
-					buckled_mob.pixel_y = 4
-				if(WEST)
-					buckled_mob.pixel_x = 2
-					buckled_mob.pixel_y = 4
+			if(buckled_mob.get_num_legs() <= 0)
```

Comment:
"if you have negative legs"
Less than 0, as in Negative.

```get_num_legs()``` will never return anything lower than 0.
This ```<=``` is most likely meant to be ```==``` to check that they have NO legs.
(however I would swap the two ```switch()```s so that it checks you have legs (```> 0```) before doing normal pixel offsets, and doing the legless ones in the ```else```